Ryuushinzou/SmartCarCleaner
# Init Spring project

Resolves #16 Spring project init

### Description
Created the root module for the **Spring** project.

### Changes
- Added multiple files related to the Spring project config.

### How has the solution been tested and validated
1. Launch **IntelliJ IDEA**.
2. Tap **Open** if the IntelliJ IDEA welcome screen is prompted; tap **File>Open** if an older project is prompted.
3. From the navigation window locate the **SmartCarCleaner** directory/folder.
4. Select the **backend/** directory/folder from the window.
5. Tap **Open**.
6. Wait for all the IDE tasks to execute.
7. Launch the **SmartCarCleanerApplication**.
8. Notice that no errors occur.

### Extra notes
- If using other IDEs, similar steps should be available; end result should be the same at steps & & 8.
